js/a11y.js

```diff
@@ -448,7 +448,8 @@ class A11y extends Backbone.Controller {
         style.visibility === 'hidden' ||
         item.getAttribute('aria-hidden') === 'true';
     });
-    if (isNotVisible) {
+    const isInDOM = Boolean($element.parents('body').length);
```

Comment:
Minor: Could this `isInDOM` check be done and early false return before the isNotVisible computation?